Automattic/jetpack
# Boost: Remove back button

Fixes #29539

## Proposed changes:

* Removed back button from purchase success page

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Check wp-admin/admin.php?page=jetpack-boost#/purchase-successful on your test site to make sure the Back button is removed.
![574Rux.png](https://user-images.githubusercontent.com/3737780/233045141-c08ece13-28d9-49ac-a227-95e9bda25125.png)
